### DISCUSSION TITLE
Value of MAX_NUMBER_EXPORT_RECORDS - how it affects performance?

### DISCUSSION THREAD

**Discussion by trererezz** [YetiForceCompany/YetiForceCRM#15813] [How to]
Hi,

is it safe to set value of 'MAX_NUMBER_EXPORT_RECORDS' from config/performance.php to 1000000?
Or would it destabilaze the application and cause troubles while using?

**Comment by rskrzypczak** [comment]
In fact, it all depends on the capabilities of the server or the size of the data.
However, the export of so many entries seems overdone.
I don't see the point in exporting such large numbers of entries at once, since you won't import them all at once either.

**Reply by trererezz** [reply]
Point is to export information about cost invoices, specifically positions from advance module in order to import to excel/powerbi

The limit is 1000 - not enough to export all data. 
I am not sure if I increase to number that is needed at the time (ex. 10k) and in future increase it again or set it to bigger value once.

My concern is that if I export huge number of records, is it possible to cause any kind of bugs (ex. action of others users might not be recorded like creating records while export is being proccessed)
